From ad8f35a55cb3f374574d4c6821a699d7dfad47f2 Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Tue, 16 Oct 2012 16:08:29 +0800
Subject: [PATCH 2173/2463] ENGR00229803-1 lcdif: export
 mxcfb_elcdif_register_mode needed by sii902x
 driver

This is needed when build sii902x hdmi driver as module

Signed-off-by: Robby Cai <R63905@freescale.com>
---
 drivers/video/mxc/mxc_elcdif_fb.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/video/mxc/mxc_elcdif_fb.c b/drivers/video/mxc/mxc_elcdif_fb.c
index 83a62db..6624fd0 100644
--- a/drivers/video/mxc/mxc_elcdif_fb.c
+++ b/drivers/video/mxc/mxc_elcdif_fb.c
@@ -586,6 +586,7 @@ void mxcfb_elcdif_register_mode(const struct fb_videomode *modedb,
 
 	return;
 }
+EXPORT_SYMBOL(mxcfb_elcdif_register_mode);
 
 int mxc_elcdif_frame_addr_setup(dma_addr_t phys)
 {
-- 
1.7.7.4

